src/service/data_products/heatmap.py

```diff
@@ -47,6 +50,11 @@
 
 _OPT_AUTH = KBaseHTTPBearer(optional=True)
 
+# Default string columns present in heatmap row data but not existing in the HeatMapMeta
+ID_COLS = [names.FLD_KBASE_ID, names.FLD_LOAD_VERSION, names.FLD_COLLECTION_ID,
```

Comment:
Do we really need anything except the ID column in this list? The rest aren't handled by arbitrary filters, they're built into the filter set and the users shouldn't be specifying those columns.

I know they're in the genomeutils specs but that's because they're used to create the view. That's not the case here

Reply:
👍

Reply:
:+1: